modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -15,8 +15,6 @@
 
```

Comment:
Title to `Query vector embeddings (vector indexes)` ?

Reply:
I will keep the current title.